server/routes/search.js

```diff
@@ -9,6 +10,14 @@ module.exports = function( req, res ){
   // input text
   var text = req.query.text || '';
 
+  // placetype filter
+  var filter = {
+      placetype: ( req.query.placetype || '' )
+                    .split(',')
+                    .map(a => a.trim())
```

Comment:
I love the much shorter function syntax!

Reply:
yeah that's really cool